### DISCUSSION TITLE
Add NDA similar EULA when someone logs i

### DISCUSSION THREAD

**Discussion by amukulsingh** [community/community#23759] [New to GitHub]
<p>We want to include NDA in Github so that if some logs in he should accept our NDA then proceed further or when he start a work then he should accept our ndA or read it then proceed further. It is one of the important things for us.</p>

**Comment by ghost** [comment, marked as answer]
<p>Hi <a class="mention" href="/amukulsingh">@amukulsingh</a> - welcome to the GitHub Support Community!</p>
<p>Thanks for your question - unfortunately this is not something that we support. If a repo is public, it’s  not possible to make the access conditional.</p>
<p>If you have code that is protected by an NDA, the best option is to make the repository private and invite users to the repository once they have satisfied your requirements.</p>
<p>I hope this helps!</p>
